cmd/do-agent/config.go

```diff
@@ -94,6 +96,12 @@ func init() {
 	kingpin.Flag("k8s-metrics-path", "enable DO Kubernetes metrics collection (this must be a DOKS metrics endpoint)").
 		StringVar(&config.kubernetes)
 
+	kingpin.Flag("bearer-token", "sets the `Authorization` header on every scrape request with the configured bearer token").
+		StringVar(&config.bearerToken)
+
```

Comment:
should we enforce that they only set one of the 2 options possibly to avoid confusion when someone sets both?